Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="246" judge-id="95" status="SE" scoring="ACM" run-tests="1" contest-id="3202" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>2d3fb7bb-2a10-441d-962f-262c87316f0b</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="281" real-time="300" max-memory-used="18698240" >
  <program-stats-str>{ utime=269, stime=12, ptime=281, rtime=300, maxvsz=18698240, maxrss=19349504, nvcsw=1, nivcsw=39 }</program-stats-str>
      <input size="2">3
</input>
      <output size="0"></output>
      <stderr size="266">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000246&quot;, line 4, in &lt;module&gt;
    v
NameError: name 'v' is not defined
</stderr>
    </test>
  </tests>
</testing-report>
